tests/integration/lib/StackTest.hs

```diff
@@ -149,74 +135,6 @@ stackErr args = do
   ec <- stack' args
   when (ec == ExitSuccess) $ error "stack was supposed to fail, but didn't"
 
-type Repl = ReaderT ReplConnection IO
-
-data ReplConnection = ReplConnection
-  { replStdin  :: Handle
-  , replStdout :: Handle
-  }
-
-nextPrompt :: Repl ()
-nextPrompt = do
-  (ReplConnection _ replStdoutHandle) <- ask
-  c <- liftIO $ hGetChar replStdoutHandle
-  if c == '>'
-    then do
-      -- Skip next character
-      void $ liftIO $ hGetChar replStdoutHandle
-    else nextPrompt
-
-replCommand :: String -> Repl ()
-replCommand cmd = do
-  (ReplConnection replStdinHandle _) <- ask
-  liftIO $ hPutStrLn replStdinHandle cmd
-
-replGetLine :: Repl String
-replGetLine = ask >>= liftIO . hGetLine . replStdout
-
-replGetChar :: Repl Char
-replGetChar = ask >>= liftIO . hGetChar . replStdout
```

Comment:
Even if it is not currently used, it seems to me a pity to lose `replGetChar` from the `StackTest` API.

Reply:
Well, even `nextPrompt` did not use it... and I just couldn't quite imagine a test scenario that would.

But no problem, will restore